### DISCUSSION TITLE
A web portal for managing licenses

### DISCUSSION THREAD

**Discussion by tobinharris** [keygen-sh/community#28] [General]
Do you have a solution that might allow non-technical admin users to create and distribute licenses for customers? Or can you recommend an approach?

**Comment by ezekg** [comment]
What features would you be looking for? We're currently redesigning our dashboard, to be easier to use for a larger variety of technical and non-technical users in the future. What about the current UI is hard to use for the non-technical personnel? Just so we can be sure to not introduce the same issues in the new UI.

Until then, we typically recommend building a custom UI that better fits your business's needs, backed by our API.

**Reply by tobinharris** [reply]
Thanks. 

Allowing an admin to come in and set up customers, packages of products with entitlements, and then export/share licenses. Then see % license activation. 

Possibly tweaking license entitlements after distribution, although we're not sure if licenses should be mutable or not. That's more like account-based licensing?

Tobin

**Reply by ezekg** [reply]
These are all things that are available via the admin dashboard today, aside from the % of license activation (better analytics is on our roadmap). Is there a reason the current dashboard doesn't work here?

**Reply by tobinharris** [reply]
Thanks, @ezekg. I think I need to be less lazy and take a proper look at your portal :) How do I do this, I saw a demo portal on the website but it didn't seem like the complete system on first glance?

**Reply by ezekg** [reply]
The demo is read-only. If you want full access, you'll need to sign up for an account and test it out that way.

**Reply by ezekg** [reply]
@AI-M-BOT I'm not sure I understand. You want to allow another administrator to manage your licenses, without letting them access the account? They can't manage licenses for your account without access to the account in some form. If you're wanting to limit e.g. access to billing, then you can set the administrator's role to something other than `admin`, e.g. `sales-agent`.

**Comment by fmg-pete** [comment]
Keygen's portal lets you do ANYTHING... but the number of options are pretty overwhelming. I still don't really understand what many of them do. Here's something I whipped up this week that any admin-type person could use. I'll do something similar for the customer-facing portal and probably integrate Stripe payments into it.

![image](https://user-images.githubusercontent.com/116123362/196563010-03130c73-c040-4b63-ad08-68860b010b90.png)

[a comment by an automated account is not shown]
